lookout/style/format/code_generator.py

```diff
@@ -67,14 +71,19 @@ def apply_predicted_y(self, vnodes: Sequence[VirtualNode],
         :param vnodes_y: Sequence of the labeled `VirtualNode`-s corresponding to labeled samples.\
                          Should be ordered by start position value.
         :param y_pred: The model predictions for `vnodes_y` `VirtualNode`-s.
+        :param rule_winners: Sequence of applied rules to generate y_pred.
```

Comment:
If we have `rule_winners`, we can recover `y_pred` by looking at what each rule predicts.